client/src/extension.ts

```diff
@@ -7,14 +7,21 @@ import * as vscode from "vscode";
 
 import * as CommandFn from "./commands";
 import * as Subscriptions from "./subscriptions";
-import { LibraryJson, PowerQueryApi } from "./vscode-powerquery.api";
-import { CommandConstant } from "./commandConstant";
+import { CommandConstant, ConfigurationConstant } from "./constants";
+import { LibrarySymbolClient } from "./librarySymbolClient";
+import { LibrarySymbolManager } from "./librarySymbolManager";
+import { PowerQueryApi } from "./powerQueryApi";
 
 const commands: vscode.Disposable[] = [];
+const symbolDirectoryWatchers: Map<string, vscode.Disposable> = new Map<string, vscode.Disposable>();
```

Comment:
I'm not actually seeing this being used. Are you planning future work with it, or am I missing something?

Reply:
Oops - yes, I had planned to add watchers to each symbol file directory, but at this point I'd rather leave that for a future change. I'll remove these for now.